src/py21cmmc/mcmc.py

```diff
@@ -180,8 +180,144 @@ def run_mcmc(
         ),
         **mcmc_options,
     )
-
+    
     # The sampler writes to file, so no need to save anything ourselves.
     sampler.startSampling()
+    
+    return sampler
+
+
+def run_multinest(
+    core_modules,
+    likelihood_modules,
+    params,
+    datadir=".",
+    model_name="21CMMC",
+    log_level_21CMMC=None,
+    n_live_points=100,
+    importance_nested_sampling=True,
+    sampling_efficiency=0.8,
+    evidence_tolerance=0.5,
+    max_iter=50,
+    multimodal=True,
+    write_output=True,
+    resume=True,
+    **multinest_options,
+):
+    """
+
+    Parameters
+    ----------
+    core_modules : list
+        A list of objects which define the necessary methods to be core modules (see :mod:`~py21cmmc.mcmc.core`).
+    likelihood_modules : list
+        A list of objects which define the necessary methods to be likelihood modules (see
+        :mod:`~py21cmmc.mcmc.likelihood`)
+    params : dict
+        Parameters which will be sampled by the chain. Each entry's key specifies the name of the parameter, and
+        its value is an iterable `(val, min, max, width)`, with `val` the initial guess, `min` and `max` the hard
+        boundaries on the parameter's value, and `width` determining the size of the initial ball of walker positions
+        for the parameter.
+    datadir : str, optional
+        Directory to which MCMC info will be written (eg. logs and chain files)
+    model_name : str, optional
+        Name of the model, which determines filenames of outputs.
+    log_level_mcmc : (int or str, optional)
+        The logging level of the cosmoHammer log file.
+        for details. This setting affects only the file output by the MCMC run.
+    log_level_mcmc_stream : (int or str, optional)
+        The logging level of the stdout/stderr stream from cosmoHammer. This has the same output as the cosmoHammer
+        log file, but is printed to screen. See `log_level_mcmc` for input specifications.
+    log_level_21CMMC : (int or str, optional)
+        The logging level of the 21cmFAST Python code (specifically the "21CMMC" logging object). By default, this
+        logger has only a stdout handler. See https://docs.python.org/3/library/logging.html#logging-levels
+
+    Essential MultiNest Parameters
```

Comment:
The parameters here should be added to the ``Parameters`` section (as noted in previous comment)

Reply:
I don't think I understand this. These parameters are taken directly from MultiNest, which should be specified when calling run_multinest. Therefore I consider them as default Parameters, same as core_modules, likelihood_modules, etc. Putting them in a second section (Essential MultiNest Parameters) is just showing that they are from MultiNest.

Are you suggesting removing the title (Essential MultiNest Parameters) and put these parameters in the "Parameter" section, or putting these parameters in **essential_multinest?

Reply:
You can put them in a "Other Parameters" section -- remember that these docstrings are parsed by a program to convert to html documentation. The parser knows about the Parameters section and an "Other Parameters" section, and can format them appropriately.